persistence-modules/spring-boot-persistence-2/README.md

```diff
@@ -1,3 +1,4 @@
 ### Relevant Articles:
 
 - [Using JDBI with Spring Boot](https://www.baeldung.com/spring-boot-jdbi)
+- [Oracle Connection Pooling With Spring](https://www.baeldung.com/oracle-connection-pooling-with-spring)
```

Comment:
Don't add your article to the readme. We have a post-processing step that handles this.

Reply:
Removed from readme